### PR TITLE
Move wideHeader [] strings into handlerEntry

### PR DESCRIPTION
**Pull request by shiywang**
To adress this problem: https://github.com/openshift/origin/pull/12354, make `-o wide` more flexible to be used by OpenShift

### CODE REVIEW

File: pkg/kubectl/resource_printer.go [hunk 1]
```diff
@@ -414,7 +415,7 @@ func (h *HumanReadablePrinter) EnsurePrintWithKind(kind string) {
 
 // Handler adds a print handler with a given set of columns to HumanReadablePrinter instance.
 // See validatePrintHandlerFunc for required method signature.
-func (h *HumanReadablePrinter) Handler(columns []string, printFunc interface{}) error {
+func (h *HumanReadablePrinter) Handler(columns []string, wideColumns []string, printFunc interface{}) error {
```
Comment (liggitt) [inline pkg/kubectl/resource_printer.go]: name the arg to make clear the wide columns are added to the normal columns in wide mode (they don't need to repeat columns in the two lists)

File: pkg/kubectl/resource_printer.go [hunk 2]
```diff
@@ -2351,38 +2366,9 @@ func formatLabelHeaders(columnLabels []string) []string {
 }
 
 // headers for -o wide
-func formatWideHeaders(wide bool, t reflect.Type) []string {
+func formatWideHeaders(wide bool, wideColumns []string) []string {
```
Comment (liggitt) [inline pkg/kubectl/resource_printer.go]: remove this function... this isn't useful any more

File: pkg/kubectl/resource_printer.go [hunk 3]
```diff
@@ -515,6 +517,19 @@ var (
 	clusterColumns                   = []string{"NAME", "STATUS", "AGE"}
 	networkPolicyColumns             = []string{"NAME", "POD-SELECTOR", "AGE"}
 	certificateSigningRequestColumns = []string{"NAME", "AGE", "REQUESTOR", "CONDITION"}
+
+	//this is columns for -o wide
```
Comment (liggitt) [inline pkg/kubectl/resource_printer.go]: I would group these by object, so you can see the normal and wide columns for each type together

**Comment by pwittrock** [comment]
@liggitt Mind if I reassign to you since you have already started reviewing?

**Comment by liggitt** [comment]
> @liggitt Mind if I reassign to you since you have already started reviewing?

not at all

File: pkg/kubectl/resource_printer.go (hunk 1 above)
Comment (shiywang) [inline pkg/kubectl/resource_printer.go]: still not sure about here if it's what you want, ptal

**Comment by liggitt** [comment]
@k8s-bot ok to test

**Comment by shiywang** [comment]
@liggitt PTAL

**Pull request merged**